tests/testsuite/namespace.go

```diff
@@ -249,6 +251,28 @@ func CleanNamespaces() {
 	}
 }
 
+// waitForNamespaceSCCAnnotations waits up to 30s for the cluster-policy-controller to add the SCC related
+// annotations to the provided namespace.
+func waitForNamespaceSCCAnnotations(c kubecli.KubevirtClient, nsName string) {
+	EventuallyWithOffset(1, func() error {
+		ns, err := c.CoreV1().Namespaces().Get(context.Background(), nsName, metav1.GetOptions{})
+		if err != nil {
+			return err
+		}
+		if ns.Annotations == nil {
+			return fmt.Errorf("namespace %q annotaions are nil", nsName)
+		}
+		for k := range ns.Annotations {
+			// annotations to check based off of
+			// https://github.com/openshift/cluster-policy-controller/blob/master/pkg/security/controller/namespace_scc_allocation_controller.go#L141
+			if k == securityv1.UIDRangeAnnotation {
+				return nil
+			}
+		}
+		return fmt.Errorf("namespace %q does not have SCC annotaions", nsName)
+	}, 250*time.Millisecond, 30*time.Second).ShouldNot(HaveOccurred(), fmt.Sprintf("namespace %q should eventually have SCC annotations %q", nsName, securityv1.UIDRangeAnnotation))
```

Comment:
Is 30 second empiric ?
Asking so we will have memory of how it was selected (margins are fine of course)